app/models/version.rb

```diff
@@ -347,10 +351,6 @@ def authors_format
     errors.add :authors, "must be an Array of Strings"
   end
 
-  def update_prerelease
```

Comment:
This is used in `before_save` callback. You shold remove the `before_save` callback as well (it is reason to have red CI now).